### PR TITLE
Documentation Improvements from UML

### PR DESCRIPTION
**Pull request by jmirtsch**
Import of documentation for some constants and property constants from UML

### CODE REVIEW

File: IFC4x3/PropertyConstants/c/CONST_0JfPhvghHD3h$pIZ_hcMEX.xml [hunk 1]
```diff
@@ -1,3 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="CONST_0JfPhvghHD3hpIZ_hcMEX" Name="CONST" UniqueId="13a59af9-aab4-4d0e-bff3-4a3fab9963a1" />
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="CONST_0JfPhvghHD3hpIZ_hcMEX" Name="CONST" UniqueId="13a59af9-aab4-4d0e-bff3-4a3fab9963a1">
+	<Documentation>Specifies that transition of the width between the previous width event and this event shall be constant and equal to the previously specified width meaning that the change in width shall occur instantaneously at the location of this event.</Documentation>
```
Comment (SergejMuhic) [inline IFC4x3/PropertyConstants/c/CONST_0JfPhvghHD3h$pIZ_hcMEX.xml:3]: This does not make sense. First, it says that the transition of the width is constant and finishes by saying that the change in width occurs instantaneously. It was probably meant that not the transition is constant but the width remains constant until it reaches the event in question.

File: IFC4x3/PropertyConstants/l/LINEAR_1m2i15jzb7DeT8TSDmT03v.xml [hunk 2]
```diff
@@ -1,3 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="LINEAR_1m2i15jzb7DeT8TSDmT03v" Name="LINEAR" UniqueId="700ac045-b7d9-4736-8748-75c3707400f9" />
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="LINEAR_1m2i15jzb7DeT8TSDmT03v" Name="LINEAR" UniqueId="700ac045-b7d9-4736-8748-75c3707400f9">
```
Comment (SergejMuhic) [inline IFC4x3/PropertyConstants/l/LINEAR_1m2i15jzb7DeT8TSDmT03v.xml:2]: Could this one and the next one be merged?

**Review by SergejMuhic — COMMENTED** [review]
A couple of questions directly on the parts that were a bit ambiguous.

**Review by AlexBrad1eyCT — COMMENTED** [review]
Agree with sergej's comments, other than that LFTM!

**Pull request closed without merging**